libraries/AdaptiveExpressions/ExpressionType.cs

```diff
@@ -114,17 +114,18 @@ public static class ExpressionType
         public const string DateTimeDiff = "dateTimeDiff";
 
         // Timex 
-        public const string IsDefinite = "isDefinite";
-        public const string IsTime = "isTime";
-        public const string IsDuration = "isDuration";
-        public const string IsDate = "isDate";
-        public const string IsTimeRange = "isTimeRange";
-        public const string IsDateRange = "isDateRange";
-        public const string IsPresent = "isPresent";
-        public const string GetNextViableDate = "getNextViableDate";
-        public const string GetPreviousViableDate = "getPreviousViableDate";
-        public const string GetNextViableTime = "getNextViableTime";
-        public const string GetPreviousViableTime = "getPreviousViableTime";
+        public const string IsDefinite = "timex.isDefinite";
+        public const string IsTime = "timex.isTime";
+        public const string IsDuration = "timex.isDuration";
+        public const string IsDate = "timex.isDate";
+        public const string IsTimeRange = "timex.isTimeRange";
+        public const string IsDateRange = "timex.isDateRange";
+        public const string IsPresent = "timex.isPresent";
+        public const string GetNextViableDate = "timex.getNextViableDate";
+        public const string GetPreviousViableDate = "timex.getPreviousViableDate";
+        public const string GetNextViableTime = "timex.getNextViableTime";
+        public const string GetPreviousViableTime = "timex.getPreviousViableTime";
+        public const string TimexResolve = "timex.resolve";
```

Comment:
Not sure how I feel about putting these all in a namespace.  We have not done this anywhere else.  Was @vishwacsena involved?

Reply:
yes, this is based on @vishwacsena's suggestion, i'm OK with both, with or without namespace.  I feel like at least namespace help distinguish "timex.resolve"

Reply:
I talked to Vishwac and we agreed that we should remove all of the timex prefixes.  We don't do this for any other data types, like string or object so we should not do it here.

---
In reply to: [486108530](https://github.com/microsoft/botbuilder-dotnet/pull/4573#discussion_r486108530) [](ancestors = 486108530)

Reply:
ok, let's do it